spec/lib/jekyll/menus_spec.rb

```diff
@@ -57,7 +56,7 @@ def page_with_menu_definition
 
         def page_with_menu_reference 
             double(
-                :data => {'menus' => ['header'], 'ext' => 'ext'}, 
+                :data => {'menus' => [ 'header' ], 'ext' => 'ext'}, 
```

Comment:
Please use double quotes, and spread hashes!

Reply:
Re: double quotes, sure.  
Re: spread hashes, I don't know what you mean.  It sounds like destructuring, and Googling "ruby spread hashes" brings back a lot of results on splats/double splats, but I don't know how that's applicable here.  Can you provide a reference/example?

Reply:
Do you mean inserting whitespace inside the braces?
```
tight = {a:1}
spread = { a: 1 }
```

Reply:
@envygeeks RFC ^

Reply:
Spread hashes are

```ruby
hash = {
  hello: :world,
}
```